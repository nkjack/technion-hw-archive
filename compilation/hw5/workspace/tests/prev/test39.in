int Fibonacci(int length) {
	int first = 1;
	int second = 1;
	int temp;
	while (length > 2) {
		temp = second;
		second = first + second;
		first = temp;
		length = length - 1;
	}
	return second;
}
void maxTestsTestFibonacci() {
	printi(Fibonacci(3));
	print("\n");
	printi(Fibonacci(5));
	print("\n");
	printi(Fibonacci(10));
}

void main() {
     maxTestsTestFibonacci();
     print("\nI hate this course\n");
}
