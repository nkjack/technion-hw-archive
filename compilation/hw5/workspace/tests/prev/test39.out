Loaded: ./exceptions.s
2
5
55
I hate this course
